dataactbroker/README.md

```diff
@@ -240,67 +240,84 @@ Example Output:
-  "agency_name":"Name of the agency",
-  "reporting_period_start_date":"03/31/2016",
-  "reporting_period_end_date":"03/31/2016",
-  "is_quarter":False,
-  "existing_submission_id: 7 (leave out if not correcting an existing submission)
-}
-```
+This route will upload the files, then kick off the validation jobs. It will return the submission id, which can be used for the `/v1/check_status/` route to poll for validation completion. 
 
-Example output:
+**NOTE: If using the frontend, you will need to call /v1/finalize_job/ to kick off validation once upload is complete**
 
-```json
-{
-  "submission_id": 12345,
 
-  "bucket_name": "S3-bucket",
+#### Additional Required Headers:
 
-  "award_id": 100,
-  "award_key": "2/1453474323_awards.csv",
+- `Content-Type` - `"multipart/form-data"`
 
-  "appropriations_id": 101,
-  "appropriations_key": "2/1453474324_appropriations.csv",
+#### Request Parameters:
+- `appropriations` - local path to file using @ notation
+- `program_activity` - local path to file using @ notation
+- `award_financial` - local path to file using @ notation
+- `cgac_code` - **required if not FREC** string, CGAC of agency (null if FREC agency)
+- `frec_code` - **required if not CGAC** string, FREC of agency (null if CGAC agency)
+- `is_quarter` - boolean (true for quarterly submissions)
+- `reporting_period_start_date` - string, starting date of submission (MM/YYYY)
+- `reporting_period_end_date` - string, ending date of submission (MM/YYYY)
 
-  "award_financial_id": 102,
-  "award_financial_key": "2/1453474327_award_financial.csv",
+**NOTE**: for monthly submissions, start/end date are the same
 
-  "program_activity_id": 103,
-  "program_activity_key": "2/1453474333_program_activity.csv",
+#### Example curl request:
+```
+    curl -i -X POST 
+        -H "x-session-id: abcdefg-1234567-hijklmno-89101112"  
+        -H "Content-Type: multipart/form-data" 
+        -F 'cgac_code=020' 
+        -F 'frec_code=null' 
+        -F 'is_quarter=true' 
+        -F 'reporting_period_start_date=04/2018' 
+        -F 'reporting_period_end_date=06/2018' 
+        -F "appropriations=@/local/path/to/a.csv" 
+        -F "award_financial=@/local/path/to/c.csv"  
+        -F "program_activity=@/local/path/to/b.csv"
+    /v1/submit_files/
 
-  "credentials": {
-    "SecretAccessKey": "ABCDEFG",
-    "SessionToken": "ABCDEFG",
-    "Expiration": "2016-01-22T15:25:23Z",
-    "AccessKeyId": "ABCDEFG"
-  }
+
+```
+
+#### Example output:
+
+```json
+{
+  "success":"true",
+  "submission_id": 123
 }
 ```
 
 #### POST "/v1/upload_detached_file/"
-A call to this route should be of content type `"multipart/form-data"`, and should use @ notation for the value of the "fabs" key, to indicate the local path to the file to be uploaded.
+A call to this route should be of content type `"multipart/form-data"`, and should use @ notation for the value of the "fabs" key, to indicate the local path to the file to be uploaded. 
```

Comment:
This is just for the API, be sure to specify that.

Reply:
Done